test_async_outline_vpn.py

```diff
@@ -0,0 +1,81 @@
+import os
+import pytest
+import pytest_asyncio
+from outline_vpn.outline_vpn_async import AsyncOutlineVPN
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
I'd rather not use this since we are not tracking .env files. All the data that we use for testing should come from the installation logs.